src/components/Dialog/DialogContainer.stories.tsx

```
@@ -1,16 +1,138 @@
+} satisfies Meta<typeof CustomDialog>;
 
 export default meta;
 type Story = StoryObj<typeof meta>;
 
-export const Basic: Story = {
-  render: () => <DialogContainer />,
+export const Alert: Story = {
+  render: () => (
+    <CustomDialog>
+      <CustomDialog.Trigger>
+        <button className="typography-body-b bg-secondary-negative cursor-pointer rounded-[5px] px-4 py-3 text-white">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)